kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/LocallyRunningOperatorRbacHandler.java

```diff
@@ -235,6 +268,138 @@ public void afterAll(ExtensionContext context) throws Exception {
         testActorClient.close();
     }
 
+    public static class ResourceBuilder<SPEC_BUIlDER extends VisitableBuilder<SPEC, SPEC_BUIlDER>, SPEC extends Editable<SPEC_BUIlDER>, RESOURCE extends CustomResource<SPEC, ?>> {
+
+        private final RESOURCE resource;
+        private final TestActor testActor;
+
+        public ResourceBuilder(RESOURCE emptyResource, ObjectMeta meta, TestActor testActor) {
+            this.resource = emptyResource;
+            this.testActor = testActor;
+            this.resource.setMetadata(meta);
```

Comment:
Why doesn't the resource already have the right metadata?

Reply:
it does, I'll look for a tidier way. I wanted to have one method for create and update. We want to create metadata in the 'create' case, but probably not the `update` case.